CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## 8.0.0-alpha.1
 
-- The first public preview of [Polly v8](https://github.com/App-vNext/Polly/issues/1048) with our [new high-performance core API](https://github.com/App-vNext/Polly/blob/main/src/Polly.Core/README.md) and extensions - Thanks to:
+- The first public preview of [Polly v8](https://github.com/App-vNext/Polly/issues/1048) with our [new high-performance core API](https://github.com/App-vNext/Polly/blob/main/src/Polly.Core/README.md) and extensions. Visit the [samples](samples/) to see Polly V8 in action.
+- Releasing [`Polly.Core`](https://nuget.org/packages/Polly.Core) NuGet package.
```

Comment:
@martincostello Do you think we can sync this with https://github.com/App-vNext/Polly/releases/tag/8.0.0-alpha.1

(including some missing contributors)

Reply:
The changelog isn't in the NuGet package itself, so it's fine to add content to this after-the-fact rather than having to delete the release and tag and re-do it.

How about changing the structure to something like:

```
- The first release of the new NuGet packages:
  - [Polly.Core](link)
  - [etc](etc)
```

Reply:
Sure, works too!